### PR TITLE
XML removal (part 7)

### PR DESCRIPTION
**Pull request by fgalan**
Part of #1862

### CODE REVIEW

File: test/functionalTest/cases/0000_ipv6_support/ipv4_only.test.DISABLED [hunk 1]
```diff
@@ -0,0 +1,156 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0000_ipv6_support/ipv4_only.test.DISABLED]: Note this .test has been renamed with .DISABLED. Have a look to https://github.com/telefonicaid/fiware-orion/issues/1954 to know the reason.

File: test/functionalTest/cases/0000_ipv6_support/ipv6_only.test.DISABLED [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Copyright 2013 Telefonica Investigacion y Desarrollo, S.A.U
```
Comment (fgalan) [inline test/functionalTest/cases/0000_ipv6_support/ipv6_only.test.DISABLED]: Note this .test has been renamed with .DISABLED. Have a look to https://github.com/telefonicaid/fiware-orion/issues/1954 to know the reason.

**Comment by kzangeli** [comment]
LGTM (leido en diagonal)

**Comment by crbrox** [comment]
LGTM

**Pull request merged**